docs-dev/e2e-tests.md

```diff
@@ -42,7 +42,7 @@ az role assignment create \
 Next, the identity of this SP and the name of this resource group must be set in GitHub secrets on the repo:
 
 - `AZURE_TENANT_ID`: The `tenant` property from the `az ad sp create-for-rbac` output.
-- `AZURE_USERNAME`: The `AZURE_SP_NAME` variable in the script.
+- `AZURE_USERNAME`: The `appId` property from the `az ad sp create-for-rbac` output.
```

Comment:
Name did not work for me. According to docs, it should be app id instead. https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli#sign-in-using-a-service-principal

Reply:
Name works for me.

```sh
$ az login --service-principal --tenant '<guid>' --username 'http://iot-identity-service-e2e-tests' '--password=<password>'
```

Reply:
Hm, it did not work when I put this name as GH Secret. I made sure I used `http://iot-identity-service-e2e-tests`. I think that using appId is better, anyways, unless you have strong preferences.

Reply:
No that's fine, I was just curious. Changing it to `.appId` is fine.

You also need to change line 81 in line with lines 78 and 79 (`...` and the comment).

Reply:
Done